doc/release-notes.md

```diff
@@ -70,6 +71,67 @@ This adds operations to the `/services`, `/identities` and `/edge-routers` endpo
        * Query related identities: GET /services/<service-id>/identities?filter=<optional-filter>
        * Query related edge routers: GET /services/<service-id>/edge-routers?filter=<optional-filter>
 
+## Filtering Entity Lists by Roles
+When building UIs it may be useful to list entities which have role attributes by role filters, to see what policy changes may look like.
+
+     * Endpoint: /edge-routers
+     * Query: GET /edge-routers now supports two new query paramters
+         * roleFilter. May be specified more than one. Should be an id or role specifier (ex: @38683097-2412-4335-b056-ae8747314dd3 or #sales)
+         * roleSemantic. Optional. Defaults to AllOf if not specified. Indicates which semantic to use when evaluating role matches
+ 
+     * Endpoint: /identities
+     * Query: GET /identities now supports two new query paramters
+         * roleFilter. May be specified more than one. Should be an id or role specifier (ex: @38683097-2412-4335-b056-ae8747314dd3 or #sales)
+         * roleSemantic. Optional. Defaults to AllOf if not specified. Indicates which semantic to use when evaluating role matches
+ 
+     * Endpoint: /services
+     * Query: GET /services now supports two new query paramters
+         * roleFilter. May be specified more than one. Should be an id or role specifier (ex: @38683097-2412-4335-b056-ae8747314dd3 or #sales)
+         * roleSemantic. Optional. Defaults to AllOf if not specified. Indicates which semantic to use when evaluating role matches
+         
+Note that a roleFilter should have one role specifier (like `@some-id` or `#sales`). If you wish to specify multiple, provide multiple role filters. 
+
+    /edge-routers?roleFilter=#sales@roleFilter=#us
```

Comment:
Shouldn't this be `?roleFilter=<val>&roleFilter=<val>` Looks like a @ instead of an &

Reply:
Fixed